test/artillery/artillery-acceptance.js

```diff
@@ -0,0 +1,17 @@
+const sampling = require('./sampling.js')
```

Comment:
Maybe it's just me, but perhaps I could recommend that you can make a quick comment or something that summarizes what artillery-acceptance, artillery-performance, and artillery monitoring does?  I mean, I guess the titles are descriptive enough....maybe I am just being nitpicky.